variation/validate.py

```diff
@@ -0,0 +1,113 @@
+"""Module for Validation."""
+from cool_seq_tool.data_sources import SeqRepoAccess, TranscriptMappings, UTADatabase
+from gene.query import QueryHandler as GeneQueryHandler
+
+from variation.schemas.classification_response_schema import Classification
+from variation.schemas.validation_response_schema import ValidationSummary
+from variation.validators import (
+    Amplification,
+    CdnaDeletion,
+    CdnaDelIns,
+    CdnaInsertion,
+    CdnaReferenceAgree,
+    CdnaSubstitution,
+    GenomicDeletion,
+    GenomicDeletionAmbiguous,
+    GenomicDelIns,
+    GenomicDuplication,
+    GenomicDuplicationAmbiguous,
+    GenomicInsertion,
+    GenomicReferenceAgree,
+    GenomicSubstitution,
+    ProteinDeletion,
+    ProteinDelIns,
+    ProteinInsertion,
+    ProteinReferenceAgree,
+    ProteinStopGain,
+    ProteinSubstitution,
+)
+
+
+class Validate:
+    """The validation class."""
+
+    def __init__(
+        self,
+        seqrepo_access: SeqRepoAccess,
+        transcript_mappings: TranscriptMappings,
+        uta: UTADatabase,
+        gene_normalizer: GeneQueryHandler,
+    ) -> None:
+        """Initialize the validate class.
+
+        :param seqrepo_access: Access to SeqRepo data
+        :param transcript_mappings: Access to transcript mappings
+        :param uta: Access to UTA queries
+        :param gene_normalizer: Access to gene-normalizer
+        """
+        params = [seqrepo_access, transcript_mappings, uta, gene_normalizer]
+        self.validators = [
```

Comment:
this order looks intentional -- it'd be good to throw a quick description into the docstring describing the general idea of what that order is (e.g. so that if someone else wants to add a new validator they could get a good idea of where to put it)

Reply:
@korikuzma realizing now that this order is consistent across the other steps (ie translate) as well, so it might be good to explain it in one place and reference that from the other places that it's used

Reply:
Added in latest commit. Let me know if that sounds okay or if I need to explain more